app/controllers/rentals_controller.rb

```diff
@@ -3,9 +3,12 @@ def create
   end
 
   def index
+    @rentals = Rental.where(user: current_user)
   end
 
   def show
+    @rental = Rental.find(params[:id])
```

Comment:
soucis de sécurité potentiel
```suggestion
    @rental = Rental.where(user: current_user).find(params[:id])
```

Reply:
ok done